src/components/Home.js

```diff
@@ -0,0 +1,61 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { showCountries } from '../redux/home/home';
+import '../Home.css';
+
+const Home = () => {
+
+  const southern = [
+    'Angola',
+    'Botswana',
+    'Lesotho',
+    'Mozambique',
+    'Namibia',
+    'South Africa',
+    'Swaziland',
+    'Zambia',
+    'Zimbabwe',
+  ];
+
+  const state = useSelector((state) => state.country.filter((country) => southern.includes(country.name)));
+  const dispatch = useDispatch();
+  console.log(state);
+
+  useEffect(() => {
+    dispatch(showCountries());
+  }, []);
+
+  const handleRedirect = (country) => {
+    window.location.href = `/${country}`;
+  };
```

Comment:
- [ ] Please make sure that the navigation works fine on Netlify. Currently, when the user clicks on a country, the page shows an error. (You can see it below)
![image](https://user-images.githubusercontent.com/62290545/169049350-0c618150-505b-4785-b9c9-e98e8c9e1967.png)

- [HINT] On Netlify, if you change the url, it shows that error. You can try using the [useNavigate](https://reactrouterdotcom.fly.dev/docs/en/v6/hooks/use-navigate) hook.

Reply:
Thank you for the hint! I have applied the useNavigate hook.